common-npm-packages/docker-common/dockercommandutils.ts

```diff
@@ -301,13 +301,17 @@ function parseHistoryForLayers(input: string) {
 
 function parseHistoryForV1Name(topHistoryLayer: string): string {
     let v1Name = "";
-    const layerIdString = "layerId:sha256:";
-    const indexOfLayerId = topHistoryLayer.indexOf(layerIdString);
+    let layerIdString = "layerId:sha256:";
+    let indexOfLayerId = topHistoryLayer.indexOf(layerIdString);
+    if (indexOfLayerId < 0) {
+        layerIdString = "layerId:"
+        indexOfLayerId = topHistoryLayer.indexOf(layerIdString);
+    }
     if (indexOfLayerId >= 0) {
         v1Name = topHistoryLayer.substring(indexOfLayerId + layerIdString.length);
     }
 
-    return v1Name;
+    return v1Name && v1Name !== "<missing>" ? v1Name : "";
```

Comment:
Where this "missing" literal comes from?

Reply:
Sometimes layer id will be come as <missing> in that case we need to skip that

Reply:
But why should we handle this case with returning empty string?
`v1Name : ""`

Reply:
I think I get it.